dsp-value-objects/src/test/scala/dsp/valueobjects/GroupSpec.scala

```diff
@@ -0,0 +1,114 @@
+  )
+
+  def spec = (groupNameTest + groupDescriptionsTest + groupStatusTest + groupSelfJoinTest)
+
+  private val groupNameTest = suite("GroupSpec - GroupName")(
+    test("pass an empty value and throw an error") {
+      assertTrue(GroupName.make("") == Validation.fail(V2.BadRequestException(GroupErrorMessages.GroupNameMissing)))
+      assertTrue(
+        GroupName.make(Some("")) == Validation.fail(V2.BadRequestException(GroupErrorMessages.GroupNameMissing))
+      )
+    } +
+      test("pass an invalid value and throw an error") {
+        assertTrue(
+          GroupName.make(invalidName) == Validation.fail(
+            V2.BadRequestException(GroupErrorMessages.GroupNameInvalid)
+          )
+        )
+        assertTrue(
+          GroupName.make(Some(invalidName)) == Validation.fail(
+            V2.BadRequestException(GroupErrorMessages.GroupNameInvalid)
+          )
+        )
+      } +
+      test("pass a valid value and successfully create value object") {
+        assertTrue(GroupName.make(validName).toOption.get.value == validName)
+        assertTrue(GroupName.make(Option(validName)).getOrElse(null).get.value == validName)
+      } +
+      test("pass None") {
+        assertTrue(
+          GroupName.make(None) == Validation.succeed(None)
+        )
+      }
+  )
+
+  private val groupDescriptionsTest = suite("GroupSpec - GroupDescriptions")(
+    test("pass an empty object and throw an error") {
+      assertTrue(
+        GroupDescriptions.make(Seq.empty) == Validation.fail(
+          V2.BadRequestException(GroupErrorMessages.GroupDescriptionMissing)
+        )
+      )
+      assertTrue(
+        GroupDescriptions.make(Some(Seq.empty)) == Validation.fail(
+          V2.BadRequestException(GroupErrorMessages.GroupDescriptionMissing)
+        )
+      )
+    } +
+      test("pass an invalid object and throw an error") {
+        assertTrue(
+          GroupDescriptions.make(invalidDescription) == Validation.fail(
+            V2.BadRequestException(GroupErrorMessages.GroupDescriptionInvalid)
+          )
+        )
+        assertTrue(
+          GroupDescriptions.make(Some(invalidDescription)) == Validation.fail(
+            V2.BadRequestException(GroupErrorMessages.GroupDescriptionInvalid)
+          )
+        )
+      } +
+      test("pass a valid object and successfully create value object") {
+        assertTrue(GroupDescriptions.make(validDescription).toOption.get.value == validDescription)
+        assertTrue(GroupDescriptions.make(Option(validDescription)).getOrElse(null).get.value == validDescription)
+      } +
+      test("pass None") {
+        assertTrue(
+          GroupDescriptions.make(None) == Validation.succeed(None)
+        )
+      }
+  )
+
+  private val groupStatusTest = suite("GroupSpec - GroupStatus")(
+    test("pass a valid object and successfully create value object") {
+      assertTrue(GroupStatus.make(true).toOption.get.value == true)
```

Comment:
Do you need the `== true` when using assertTrue?

Reply:
In fact, it isn't required but if I should be consistent, in the next line instead of `== false` I should use `!` in front of checked value, which is IMO less readable.